entity-types/uninstrumented-redisnode/definition.yml

```diff
@@ -0,0 +1,6 @@
+domain: UNINSTRUMENTED
+type: REDISNODE
```

Comment:
I don't think we need to do a differentiation between nodes and clusters.
We could follow the same approach than for databases:

- https://github.com/newrelic/entity-definitions/blob/main/entity-types/uninstrumented-database/definition.yml
- https://github.com/newrelic/entity-definitions/blob/main/relationships/candidates/DATABASE.yml
- https://github.com/newrelic/entity-definitions/blob/main/relationships/synthesis/APM-APPLICATION-to-DATABASE.yml

Reply:
From the APM perspective, it talks to Redis. The detail of clusters vs nodes (and different clouds in the future) is something that we can abstract in the same way we did for databases.

Reply:
Hi @javimb I have update the files accordingly, Please review.ThankYou